examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -411,10 +417,37 @@ def act(self, obs, env):
         human_pos = env._sim.agents_mgr[
             1 - self._agent_idx
         ].articulated_agent.base_transformation.translation
+        human_rot = env._sim.agents_mgr[
+            1 - self._agent_idx
+        ].articulated_agent.base_rot
         robot_pos = env._sim.agents_mgr[
             self._agent_idx
         ].articulated_agent.base_transformation.translation
 
+        # Check if the human walks or rotates
+        is_human_walk = (
+            np.linalg.norm(
+                np.array((self._prev_human_pos - human_pos))[[0, 2]]
```

Comment:
It seems possible in some edge case (perhaps some future edge case) that `_prev_human_pos` will be `None` here. You should handle that gracefully; similar for `_prev_human_rot`.

Reply:
Hi Eric, I did add `if self._prev_human_pos is not None` to check the condition. Please let me know if this is okay. Thank you!